engine/modelc/src/test/test_model.cpp

```diff
@@ -93,6 +93,27 @@ TEST(ModelGLTF, LoadSkeleton)
     free(mem);
 }
 
+// See https://github.com/KhronosGroup/glTF-Asset-Generator for assets
+// https://raw.githubusercontent.com/KhronosGroup/glTF-Sample-Models/master/2.0/VertexColorTest/glTF-Embedded/VertexColorTest.gltf
+
+TEST(ModelGLTF, VertexColor3Float)
+{
+    // Model courtesy of Artsion Trubchyk (https://github.com/aglitchman), public domain
+    const char* path = "./src/test/assets/primitive_vertex_color/vertexcolor_rgb3.gltf";
+    uint32_t file_size = 0;
+    void* mem = dmModelImporter::ReadFile(path, &file_size);
```

Comment:
Just a thought, but don't we usually return a result enum and pass in pointers to the functions? 🤔 

Edit: Ah, it's only used for tests..